ios-device-identifiers.json

```diff
@@ -25,19 +25,23 @@
   "iPhone9,3": "iPhone 7 (no CDMA)",
   "iPhone9,4": "iPhone 7 Plus (no CDMA)",
   "iPhone10,1": "iPhone 8",
-  "iPhone10,2": "iPhone 8 Plus",
-  "iPhone10,3": "iPhone X",
   "iPhone10,4": "iPhone 8 (no CDMA)",
+  "iPhone10,2": "iPhone 8 Plus",
   "iPhone10,5": "iPhone 8 Plus (no CDMA)",
+  "iPhone10,3": "iPhone X",
   "iPhone10,6": "iPhone X (no CDMA)",
-  "iPhone11,2": "iPhone XS",
-  "iPhone11,4": "iPhone XS Max (special model?)",
-  "iPhone11,6": "iPhone XS Max",
+  "iPhone11,2": "iPhone Xs",
+  "iPhone11,4": "iPhone Xs Max (special model?)",
+  "iPhone11,6": "iPhone Xs Max",
```

Comment:
interesting -- Apple seem to be unsure themselves how to show the `s` in iPhone XS/Xs. I think maybe in fact like the iPhone XR it's meant to be a small-caps upper case, but that doesn't make life easy in Ascii. 

<img width="672" alt="Refurbished_iPhone_XS_64GB_-_Silver__Unlocked__-_Apple" src="https://user-images.githubusercontent.com/18395/96638298-cda12e00-1317-11eb-8e03-3161a7035ec2.png">
<img width="929" alt="iPhone_Xs_and_iPhone_Xs_Max_bring_the_best_and_biggest_displays_to_iPhone_-_Apple__UK_" src="https://user-images.githubusercontent.com/18395/96638356-e0b3fe00-1317-11eb-8756-356a2e72c091.png">
<img width="1069" alt="iPhone_XS_vs_iPhone_12_vs_iPhone_12_mini_-_Apple__UK_" src="https://user-images.githubusercontent.com/18395/96638402-f5909180-1317-11eb-906e-0345122f0db7.png">
<img width="619" alt="iPhone XS_Max_vs_iPhone XR_vs_iPhone 12 mini_-_Apple" src="https://user-images.githubusercontent.com/18395/96638511-2375d600-1318-11eb-9e3c-5a778e1eb935.png">

I'm willing to accept lower case!

Reply:
Based on the image, I agree with you that it looks like the S in "XS" is a smaller, yet still uppercase S, much like the R in "XR".

Reply:
Agreed. I've changed it to XS with caps see #9